### PR TITLE
Create meeting-event.html with empty divs

### PR DESCRIPTION
**Pull request by vivianho18**
Html page that will be used to display meeting event details. We will all contribute to this html page by editing the divs relevant to our feature.

### CODE REVIEW

File: GoMeet/src/main/webapp/meeting-event.html
Comment (BipBopBoop) [inline GoMeet/src/main/webapp/meeting-event.html]: nit: styling should be put into the style sheet: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

**Review by BipBopBoop — APPROVED** [review]
Looks good, maybe let others have a look before merging/submitting :)

File: GoMeet/src/main/webapp/meeting-event.html
Comment (anna-giang) [inline GoMeet/src/main/webapp/meeting-event.html]: Actually, can you change this heading to `vote-meeting-times` and maybe the one below also to `vote-meeting-locations`? So that its clearer what will go into the divs (i.e. the voting forms).

**Review by anna-giang — APPROVED** [review]
Good to go, after the changes as mentioned above 😊 Thank you!

File: GoMeet/src/main/webapp/meeting-event.html
Comment (vivianho18) [reply GoMeet/src/main/webapp/meeting-event.html]: Fixed :)

File: GoMeet/src/main/webapp/meeting-event.html
Comment (vivianho18) [reply GoMeet/src/main/webapp/meeting-event.html]: Done :)

**Review by theresenieva — COMMENTED** [review]
Yup, lgtm ✨

**Review by theresenieva — APPROVED** [review]

**Pull request merged**
